mapbox/geocoder_test.go

```diff
@@ -130,7 +130,7 @@ const (
                -37.75522
             ]
          },
-         "address":"60",
```

Comment:
`"address":"60"` is a real life response returned from mailbox, which shouldn't be changed.